infra/lib/attendant-ide/constants.ts

```diff
@@ -43,13 +43,13 @@ const osPackages = [
   dotNetVersion,
 ];
 
+const vscodeAccessCode = "powertools-workshop"
```

Comment:
What about taking this as a `CfnPrameter` with default value fallback to this one you suggest?

This way event operators can set specific access codes for events if needed - here thinking mainly about tier 1 events.

Reply:
Can you pls review again?